client/jetpack-cloud/sections/partner-portal/company-details-form/index.tsx

```diff
@@ -231,21 +240,26 @@ export default function CompanyDetailsForm( {
 				) }
 				{ showSignupFields && (
 					<FormFieldset>
-						<FormLabel>{ translate( 'How many sites do you manage?' ) }</FormLabel>
-						<SelectControl
-							id="managed_sites"
+						<FormLabel htmlFor="managed_sites">
+							{ translate( 'How many sites do you manage?' ) }
+						</FormLabel>
+						<FormSelect
 							name="managed_sites"
-							value={ managedSites }
-							options={ [
-								{ value: '1-5', label: translate( '1-5' ) },
-								{ value: '6-20', label: translate( '6-20' ) },
-								{ value: '21-50', label: translate( '21-50' ) },
-								{ value: '51-100', label: translate( '51-100' ) },
-								{ value: '101-500', label: translate( '101-500' ) },
-								{ value: '500+', label: translate( '500+' ) },
-							] }
-							onChange={ setManagedSites }
-						/>
+							id="managed_sites"
+							// We check for an empty "managedSites" here (instead of defining it as the fallback
+							// state value) so we do not set "managed sites" to "1-5" for all older partners
+							// when they update their company details which would otherwise happen since we
+							// pass all inputs as the payload no matter what page is being used.
+							value={ managedSites.length === 0 ? '1-5' : managedSites }
+							onChange={ handleSetManagedSites }
+						>
+							<option value="1-5">{ translate( '1-5' ) }</option>
+							<option value="6-20">{ translate( '6-20' ) }</option>
+							<option value="21-50">{ translate( '21-50' ) }</option>
+							<option value="51-100">{ translate( '51-100' ) }</option>
+							<option value="101-500">{ translate( '101-500' ) }</option>
+							<option value="500+">{ translate( '500+' ) }</option>
```

Comment:
As long as we're editing this code, let's go ahead and have the labels use the proper en dash `–` instead of a hyphen `-`.

Reply:
That would change the Salesforce integration as well since it modifies them further, so I'd prefer to keep this to the Calypso code for now 😄

Reply:
Salesforce uses the value, not the translated text.

Reply:
Oooh, that's what you're saying! Yes, that could make sense :smart: